src/lrstar/PG_Parser.cpp

```diff
@@ -699,8 +699,8 @@ void  PG_Parser::init_symtab (int max_symb)
 
 void  PG_Parser::term_symtab ()
 {
-   if (symbol != NULL) delete [] symbol;
-   if (symbol != NULL) delete [] hashvec;
+   if (symbol != NULL) FREE(symbol, max_symbols);
```

Comment:
Is valgrind complaining that the allocation was done with malloc() and release with 'delete'?
Can you make that explicit in the change description?

Reply:
That's correct, mismatch allocation/deallocation.

Reply:
How hard is it to set up valgrind these days?
Should the tools in ./scripts be updated to support it?